interop-cats/shared/src/main/scala/zio/interop/CatsConcurrentEffectSyntax.scala

```diff
@@ -0,0 +1,33 @@
+package zio.interop
+
+import cats.effect.ConcurrentEffect
+import zio.{ IO, RIO, Runtime, Task, ZIO }
+
+import scala.language.implicitConversions
+
+trait CatsConcurrentEffectSyntax {
+  implicit final def ZIOConcurrentEffectOps(@deprecated("", "") zio: ZIO.type): CatsConcurrentEffectSyntax.zioOps.type =
```

Comment:
Why deprecated?

Reply:
@ghostdogpr to silence fatal warning that parameter is unused